utils/run_in_ga_wrapper.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+IMAGE=$(docker images -all --filter label=DAOS=true --quiet | tail -n 1)
+
+docker run --name build-post --mount type=tmpfs,destination=/mnt/daos,tmpfs-mode=1777 --env COMPILER $IMAGE ./utils/run_in_ga.sh
```

Comment:
(style)  line over 80 characters
(lint) Double quote to prevent globbing and word splitting. [SC2086]